lib/audit_test.pm

```diff
@@ -78,7 +80,7 @@ sub prepare_for_test {
     my (%args) = @_;
 
     # Run test case
-    assert_script_run("cd ${testdir}${testfile_tar}/audit-test/");
+    assert_script_run("cd ${test_dir}/audit-test/");
```

Comment:
Why hard code “audit-test”？

Reply:
hmm,  
 this sub directory is from clone git repo, at the same time, the previous code used the same dir.
# ll
total 16
drwx------ 1 root root  166 Sep 27 23:53 .git
-rw------- 1 root root 6444 Sep 27 23:46 .gitignore
-rw------- 1 root root 3509 Sep 27 23:46 Makefile
drwx------ 1 root root  724 Sep 27 23:46 audit-test
-rwx------ 1 root root   84 Sep 27 23:46 run_all_tests.sh